### PR TITLE
Refactor Blobs Fees Adjustments

### PR DESCRIPTION
**Pull request by gabriele-0201**

### CODE REVIEW

**Comment by gabriele-0201** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/150?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #149** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/149?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #150** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/150?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/thrumdev/blobs/150?utm_source=stack-comment).

File: sugondat-chain/runtimes/sugondat-kusama/tests/integration_test.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+use crate::fee_adjustment::*;
```
Comment (rphmeier) [inline sugondat-chain/runtimes/sugondat-kusama/tests/integration_test.rs:1]: Glob imports are bad style and should be avoided except through specifically crafted preludes.

Comment (pepyakin) [reply sugondat-chain/runtimes/sugondat-kusama/tests/integration_test.rs:1]: Maybe, as an exception, you could do it in the `use super::*` in an inline mod, but I am not a fan tbh.

**Review by pepyakin — APPROVED** [review]

File: sugondat-chain/runtimes/sugondat-kusama/tests/integration_test.rs (hunk 1 above)
Comment (gabriele-0201) [reply sugondat-chain/runtimes/sugondat-kusama/tests/integration_test.rs:1]: To be fair, I really don't know why I used a global import. However, `fee_adjustment::{BlobsLengthToFee, NextLengthMultiplier, TransactionByteFee}` are the necessary imports, so it's not a big deal 😅 I have updated the code, and there are no more global imports

**Review by rphmeier — APPROVED** [review]

**Comment by rphmeier** [comment]
### Merge activity

* **Dec 28, 3:09 PM**: @rphmeier started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/thrumdev/blobs/150).

**Pull request merged**